README.md

```diff
@@ -20,6 +20,7 @@ A GitHub action to turn a GitHub project into a self-hosted Helm chart repo, usi
 - `charts_dir`: The charts directory
 - `charts_repo_url`: The GitHub Pages URL to the charts repo (default: `https://<owner>.github.io/<project>`)
 - `skip_packaging`: This option, when populated, will skip the packaging step. This allows you to do more advanced packaging of your charts (for example, with the `helm package` command) before this action runs. This action will only handle the indexing and publishing steps.
+- `mark_as_latest`: This option, when not set to `false`, will mark the created GitHub release as 'latest'.
```

Comment:
this text is a bit off i guess, if you set to false that will not set the release to be the latest right?

Reply:
Thanks for the review! I adjusted the wording :)